### ISSUE TITLE
I cannot get version 1.11.0 running on heroku

### ISSUE THREAD

**Issue by nonfungibletunji** [Zanobo/reaction-buildpack#2]
I updated an existing running app 1.10.0 on heroku, when tried the new build pack and v 1.10.0

I get 
remote: 
remote: fatal: Not a git repository (or any parent up to mount point /tmp)
remote: Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
remote: npm WARN apollo-cache-control@0.0.9 requires a peer of graphql@^0.10.0 || ^0.11.0 || ^0.12.0 but none is installed. You must install peer dependencies yourself.
remote: npm WARN apollo-server-core@1.3.2 requires a peer of graphql@^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 but none is installed. You must install peer dependencies yourself.
remote: npm WARN apollo-tracing@0.1.3 requires a peer of graphql@^0.10.0 || ^0.11.0 || ^0.12.0 but none is installed. You must install peer dependencies yourself.
remote: npm WARN graphql-extensions@0.0.8 requires a peer of graphql@^0.10.0 || ^0.11.0 || ^0.12.0 but none is installed. You must install peer dependencies yourself.
remote: npm WARN graphql-relay@0.5.4 requires a peer of graphql@^0.5.0 || ^0.6.0 || ^0.7.0 || ^0.8.0-b || ^0.9.0 || ^0.10.0 || ^0.11.0 but none is installed. You must install peer dependencies yourself.
remote: npm WARN react-addons-test-utils@15.6.2 requires a peer of react-dom@^15.4.2 but none is installed. You must install peer dependencies yourself.
remote: npm WARN react-taco-table@0.5.1 requires a peer of react@^15.3 but none is installed. You must install peer dependencies yourself.
remote: npm WARN react-taco-table@0.5.1 requires a peer of react-dom@^15.3 but none is installed. You must install peer dependencies yourself.
remote: npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.3 (node_modules/fsevents):
remote: npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
remote: 
remote: + http-proxy@1.17.0
remote: added 30 packages in 16.082s
remote: -----> Building Meteor app with ROOT_URL: http://tribevibe.herokuapp.com
remote: 
remote: Unable to resolve some modules:
remote: 
remote:   "transliteration" in
remote: /tmp/build_506d370f14284ac956974b3b897d01ba/lib/api/helpers.js (web.browser)
remote: 
remote: If you notice problems related to these missing modules, consider running:
remote: 
remote:   meteor npm install --save transliteration
remote: 
remote: 
remote: <--- Last few GCs --->
remote: 
remote: [640:0x2aaac90]   907085 ms: Mark-sweep 1407.9 (1931.8) -> 1407.5 (1931.8) MB, 1700.6 / 0.2 ms  allocation failure GC in old space requested
remote: [640:0x2aaac90]   908709 ms: Mark-sweep 1407.5 (1931.8) -> 1407.3 (1880.8) MB, 1623.7 / 0.3 ms  last resort GC in old space requested
remote: [640:0x2aaac90]   910319 ms: Mark-sweep 1407.3 (1880.8) -> 1407.3 (1865.8) MB, 1609.8 / 1.2 ms  last resort GC in old space requested
remote: 
remote: 
remote: <--- JS stacktrace --->
remote: 
remote: ==== JS stack trace =========================================
remote: 
remote: Security context: 0x26bfa2825ee1 <JSObject>
remote:     1: /* anonymous */ [/app/tmp/cache/1.6.1/.meteor/packages/meteor-tool/.1.6.1.tf1nko.5xio++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/source-map/lib/source-node.js:~336] [pc=0x32a10a05d690](this=0x1e280d38c0e9 <JSGlobal Object>,chunk=0x1cf0d18bae19 <String[11]: "completed">,original=0x38d3e14769d1 <Object map = 0x2974c0e95f49>)
remote:     2: SourceNode_wal...
remote: 
remote: FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory
remote:  1: node::Abort() [/app/tmp/cache/1.6.1/.meteor/packages/meteor-tool/.1.6.1.tf1nko.5xio++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/bin/node]
remote:  2: 0x121a2cc [/app/tmp/cache/1.6.1/.meteor/packages/meteor-tool/.1.6.1.tf1nko.5xio++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/bin/node]
remote:  3: v8::Utils::ReportOOMFailure(char const*, bool) [/app/tmp/cache/1.6.1/.meteor/packages/meteor-tool/.1.6.1.tf1nko.5xio++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/bin/node]
remote:  4: v8::internal::V8::FatalProcessOutOfMemory(char const*, bool) [/app/tmp/cache/1.6.1/.meteor/packages/meteor-tool/.1.6.1.tf1nko.5xio++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/bin/node]
remote:  5: v8::internal::Factory::NewUninitializedFixedArray(int) [/app/tmp/cache/1.6.1/.meteor/packages/meteor-tool/.1.6.1.tf1nko.5xio++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/bin/node]
remote:  6: 0xe1c3b3 [/app/tmp/cache/1.6.1/.meteor/packages/meteor-tool/.1.6.1.tf1nko.5xio++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/bin/node]
remote:  7: v8::internal::Runtime_GrowArrayElements(int, v8::internal::Object**, v8::internal::Isolate*) [/app/tmp/cache/1.6.1/.meteor/packages/meteor-tool/.1.6.1.tf1nko.5xio++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/bin/node]
remote:  8: 0x32a108f0463d
remote: /app/tmp/buildpacks/798444e33504be6c6c26e2101627202555c04992b3e077260254dfeb0daf97c5e46e0fce127baf9ea38d79b7840f08c0423d451c40999fe463313ad40db0113d/bin/compile: line 206:   640 Aborted                 HOME=$METEOR_DIR $METEOR build $BUILD_OPTIONS --server $ROOT_URL $SERVER_ONLY_FLAG --directory $BUNDLE_DEST
remote:  !     Push rejected, failed to compile Reaction Commerce app.
remote:

**Comment by antonpolishko** [comment]
Having the same issue. One of the suggested solutions is to set TOOL_NODE_FLAGS=’–max_old_space_size=1024' but it doesn't work for me either

**Comment by nonfungibletunji** [comment]
Yeah it was building on Heroku randomly ?? Still working to figure it out 

S Tunji Turner
sightuary
@sightuary

**Comment by nonfungibletunji** [comment]
I tried npm install -g increase-memory-limit, and still not working for me.

**Comment by sgobotta** [comment]
It's working fine on my reaction v1.11.0 app though i had to fix a dependency in the package.json
```
"transliteration": "github:reactioncommerce/transliteration",  --> "transliteration": "^1.6.2",
```

@stunjiturner your is a memory issue i guess. Did you set all the required env variables? Do you have too many Shops/Packages?
In other cases really **huge** databases were stopping the booting process in my app. What is the max elements in your mongo db for any collection?

**Comment by nonfungibletunji** [comment]
No it’s something with the transliteration, what did you change ? Exactly 

S Tunji Turner
sightuary
@sightuary

**Comment by sgobotta** [comment]
Well, i did exactly as i wrote in my last message. In the package.json I changed the transliteration source from "github:reactioncommerce/transliteration" to "^1.6.2". Then the buildpack at some point automatically executes: **meteor npm install** and the transliteration dependency will be there when Reaction Commerce initialises.
I found out the issue originates in the /lib/api/helpers.js module when **lazyLoadSlugify** function tries to asynchronously fetch **transliteration**, but its not there in node_modules, either because github:reactioncommerce/transliteration doesn't exists anymore or something else.

**Comment by rsp** [comment]
@antonpolishko @sgobotta Has this issue been resolved already?

**Comment by sgobotta** [comment]
No updates as far as I know.
